### DISCUSSION TITLE
Instance Size

### DISCUSSION THREAD

**Discussion by ShaneTheDarkLord** [SKCraft/Launcher#499] [I need help!]
Is there any way to change the size of the instance list? I have looked far and wide and cant seem to find anything to be able to change it

**Comment by hedgehog1029** [comment]
The SplitPane is configured [here](https://github.com/SKCraft/Launcher/blob/master/launcher/src/main/java/com/skcraft/launcher/dialog/LauncherFrame.java#L107) - default width is 200 pixels. You can edit this to change the default size.
